src/Aspire.Hosting.AWS/Provisioning/CloudFormationStackExecutor.cs

```diff
@@ -78,6 +78,20 @@ private static (List<Tag> tags, string? existingSha256) SetupTags(Stack? existin
             tags = existingStack.Tags ?? new List<Tag>();
         }
 
+        // Add user-specified tags from context
+        foreach (var kvp in context.Tags)
```

Comment:
In an update scenario we are starting with a collection of tags filled with the existing tags would this logic work if in the AppHost I deployed with a tag and then want to remove the tag?

Reply:
ill have to test this scenario

Reply:
actually ignore this comment. my latest commit i was still having other hardcoded stuff in it. so i need to double check again